src/shared/domain/reportInteraction.ts

```diff
@@ -1,10 +1,28 @@
 import { reportInteraction as grafanaReportInteraction } from '@grafana/runtime';
 
+import { ROUTES } from '../../constants';
+
+const PROFILES_EXPLORER_PAGE_NAME = ROUTES.PROFILES_EXPLORER_VIEW.slice(1);
+
 function getCurrentPage(): string {
   const { pathname } = new URL(window.location.toString());
   return pathname.split('/').pop() || '';
 }
 
+function getExtraProperties() {
+  const page = getCurrentPage();
+  const extraProperties: Record<string, any> = { page };
+
+  if (page === PROFILES_EXPLORER_PAGE_NAME) {
+    extraProperties.explorationType = new URLSearchParams(window.location.search).get('explorationType');
+  }
+
+  return extraProperties;
+}
+
 export function reportInteraction(interactionName: string, properties?: Record<string, unknown>) {
```

Comment:
I think at some point we might want to make `interactionName` a union type of strings literals (or something similar) so we don't need to search the code to get a list of all the interaction names we have. It would be in one list here.

Something like:

```ts
type Interaction =
  'g_pyroscope_app_service_name_selected' |
  'g_pyroscope_app_profile_metric_selected' |
  ...
```